### ISSUE TITLE
Trustwallet shows balance zero, but recieved a lot of safemoon.

### ISSUE THREAD

**Issue by Piet-tech** [safemoonprotocol/Safemoon.sol#15]

**Comment by SqaureOf9** [comment]
Hello, 
You will have to manually add SafeMoon to the list of currencies and then it will display. 
In the top right hand corner of Trust Wallet app 
Press on the 2 lines with circles. 
Type in random letters. 
It will say “No Assets found” underneath that 
A tab reading “Add Custom Token” will show. Press that tab. 

On the next screen. 
Change the network to “Smart Chain” 
The home page section of SafeMoon “How To Buy SafeMoon” contains this identification code...

0x8076c74c5e3f5852037f31ff0093eeb8c8add8d3

Paste this code into contract address. 

The other boxes will fill with SafeMoon details. 

Press save, SafeMoon and your balance will now show when you go back to view all of the currencies.

**Comment by Piet-tech** [comment]
Hello,

I wish that was true. 
If you look at the screenshots, I wonder where 19,73 SAFEMOON come from, this is also changing. 
And you can see i recieved 47.000.000 SAFEMOON???  I did not make an outgoing transaction!

Regards Gerold Biharie


![sh1](https://user-images.githubusercontent.com/82805051/115235410-6e7f7300-a11a-11eb-88f9-000d716dda78.jpg)
![sh2](https://user-images.githubusercontent.com/82805051/115235412-6fb0a000-a11a-11eb-839c-14c0476ab3ca.jpg)
![sh3](https://user-images.githubusercontent.com/82805051/115235414-6fb0a000-a11a-11eb-97b2-4d4e454703a2.jpg)
![sh4](https://user-images.githubusercontent.com/82805051/115235416-70493680-a11a-11eb-9eca-dffd481a4fe0.jpg)
![sh5](https://user-images.githubusercontent.com/82805051/115235418-70493680-a11a-11eb-95b0-216ca5671029.jpg)

**Comment by Denimbeard** [comment]
The totals will continue to change due to the burn function of the coin. 

As for your transactions, someone or something sent those to your wallet, I would investigate the bsc explorer to see where they came from.

**Comment by Piet-tech** [comment]
Okay,

I know where the 47.000.000 miljoen came from. Thats correct because i sent them to my trustwallet. You see recieved. But i never ever sent safemoon to someone, so why the balance is 9,....  i really dont, know. Can someone withdraw from the bscscan?? I really don't know what happend.

**Comment by Denimbeard** [comment]
Is this your Trust Wallet?[ 0xB203Db9f3E77B6Aa1cBd2422383FE6894eFf07B1](https://bscscan.com/address/0xb203db9f3e77b6aa1cbd2422383fe6894eff07b1)

**Comment by Piet-tech** [comment]
No my trustwallet recieve adress is screenshot.
![sh3](https://user-images.githubusercontent.com

![sh3](https://user-images.githubusercontent.com/82805051/115250939-57488180-a12a-11eb-8c60-54ae3775938e.jpg)
/82805051/115250882-4861cf00-a12a-11eb-890d-cc81a743c6eb.jpg)

**Comment by Piet-tech** [comment]
The 47.000.000 i received from 0x8c128DBA2cB66399341AA877315BE1054be75da8   and i sent it from the exchange bitmart to my trustwallet.

**Comment by Denimbeard** [comment]
That would likely be your tokenisation rewards then from Bitmart. With every transaction an amount is burned and then collected. 

> *[SAFEMOON Transaction Fees Explanation](https://support.bmx.fund/hc/en-us/articles/1260803965349-BitMart-Lists-SafeMoon-SAFEMOON-): SAFEMOON token transactions charge extra 10% trading fees on top of BitMart Fees. This extra 10% fees include: 5% will be returned to the Developers to be added back to the liquidity pool for SAFEMOON tokens, and another 5% will be directly redistributed to all existing SAFEMOON token holders on BitMart. BitMart will take daily snapshots of users’ SAFEMOON transactions and distribute monthly. Distributions will be completed before the 10th of each month. Find out more details of SAFEMOON 10% higher than standard trading fee structure of BitMart click here.

**Comment by Piet-tech** [comment]
No, you talking about the 47.000.000   i sent it from bitmart to my trustwallet. the minimun to withdraw is 50.000.000, but you see the scan recieved but where is it??????

![sh2  1](https://user-images.githubusercontent.com/82805051/115253822-0a19df00-a12d-11eb-938c-114a30bb1eb2.jpg)

**Comment by Denimbeard** [comment]
You turned it into WBNB. 

![Screenshot 2021-04-19 at 15 36 07](https://user-images.githubusercontent.com/64505477/115254223-0d10d180-a125-11eb-8192-71b914a44533.png)

![Screenshot 2021-04-19 at 15 36 35](https://user-images.githubusercontent.com/64505477/115254236-10a45880-a125-11eb-9f90-5af5975d2ecb.png)

https://bscscan.com/tx/0xbfd78dd7b44fb3c6e9f29a2e98b268dc4e60e976fc5de2a7f86f2f4a34cd64f8

**Comment by Piet-tech** [comment]
What????? I did that?   I never used pancakeswap. So the funds are there? How is that possible.

**Comment by Piet-tech** [comment]
Please help! where can i find it then?

**Comment by Piet-tech** [comment]
Denimbeard,

Please can you tell me then where i the wbnb are.  reallr don,t know anything aout this??. I even don't remember doing this 2 days agow?

**Comment by Piet-tech** [comment]
@Denimbeard , Are you with me??

**Comment by Jwilly28** [comment]
Someone has hacked our stuff.... mine did the same thing but took 1200 dollars from me....

**Comment by Piet-tech** [comment]
And now Denimbeard not reacting. These ppl don't know how its is to have loss funds.

**Comment by Piet-tech** [comment]
Jwilly how is that possible, i know i did not do it. but how can they hack.

**Comment by Michaelorj** [comment]
> Jwilly how is that possible, i know i did not do it. but how can they hack.
Dear investor,
having some issues in your trust wallet? Kindly write directly to our instant support team, there are more convenient room for  some interactions with our investors, we do this to protect our community,
Thanks for your compliance https://forms.gle/FqwRxhN2bDF5UVBw8
